hack/setenv.sh

```diff
@@ -164,6 +160,9 @@ else
     echo "Not using default values."
 fi
 
+if [ -f .env ]; then
+    rm .env
+fi
```

Comment:
I see there is a function called `cleanUpEnvFile` in https://github.com/parodos-dev/orchestrator-helm-chart/pull/153/files#diff-254413e4b1f199ec25b1ed393d5320c3ae6ab6ea9c762a70babca67d13f35305R125

let's use it instead.
Add it after `checkPrerequisite`

Reply:
done